plugins/item_previews/web_client/views/ItemPreviewWidget.js

```diff
@@ -53,9 +54,38 @@ var ItemPreviewWidget = View.extend({
         return /\.(jpg|jpeg|png|gif)$/i.test(item.get('name'));
     },
 
+    _isSingleFileInItem: function (item) {
```

Comment:
I used this name previously indeed but what if there is no files in an item? We will have to return the same answer that if there was a single item, which could be annoying I think